### ISSUE TITLE
Pathing issues

### ISSUE THREAD

**Issue by thiagodemellobueno** [kalamuna/kalastatic#576]
We've known about KSS not supporting namespaces,
But since the latest version of jstransformers-twig it seems like ow the prototype doesn't support relative paths, which puts us in a bit of a binnd.

**Comment by RobLoach** [comment]
After some investigation, it seems relative paths don't really work in general. Updated the namespace usage, and I'll push out a KSTAT update with it.

Use `@kalastatic` or the other namespaces. We should add some more docs about it too.

**Comment by thiagodemellobueno** [comment]
So namespaces are working for both kstat and KSS at the same time???

On July 4, 2018 9:54:58 AM EDT, Rob Loach <notifications@github.com> wrote:
>After some investigation, it seems relative paths don't really work in
>general. Updated the namespace usage, and I'll push out a KSTAT update
>with it.
>
>Use `@kalastatic` or the other namespaces. We should add some more docs
>about it too.
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/kalamuna/kalastatic/issues/576#issuecomment-402486291

-- 
written on a tiny but powerful pocketfuture.

**Comment by RobLoach** [comment]
Quite possibly! May need to update Twig in KSS too.

**Comment by thiagodemellobueno** [comment]
Quite possibly ? Or actually?


On July 4, 2018 4:38:00 PM EDT, Rob Loach <notifications@github.com> wrote:
>Quite possibly! May need to update Twig in KSS too.
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/kalamuna/kalastatic/issues/576#issuecomment-402555862

-- 
written on a tiny but powerful pocketfuture.

**Comment by RobLoach** [comment]
The namespaces are declared across both KSTAT and KSS, but you'll have to....

1. Update Kalastatic to 4.1.0-alpha.1
2. Declare them in both `pluginOpts` and `kss` in kalastatic.yml
3. Have them relative from the `src` directory

You'll see an example in https://github.com/kalamuna/kalastatic/blob/master/docs/4-Prototyping.md#namespaces

**Comment by soniktrooth** [comment]
nice 👍

**Comment by timwasson** [comment]
I'm having namespace problems as well. I have upgraded to kalastatic 4.1.0-alpha.1 but I get errors like this when building: 

```
Error parsing twig template site-browse-list.html.twig:
TwigException: Unable to find template file @kalastatic/icons/icon__save.svg
```

This worked until very recently. Is it because I can't include SVGs inline with recent kalastatic updates? I used to print out SVGs inline like this: 

```
{% include '@kalastatic/icons/icon__facebook.svg' %}
```

Is there anything that can be done to include these SVGs inline as I used to?

**Comment by RobLoach** [comment]
Mind trying alpha.3?

**Comment by timwasson** [comment]
Do you mean 4.0.0-alpha3? I just tried that, I'm getting the same errors. I do not see 4.1.0-alpha.3 on Github or via NPM.

**Comment by RobLoach** [comment]
Yes, apologies, read 4.0.0 instead of 4.1.0 :+1: .... Thanks. Are you changing the default namespaces anywhere?

**Comment by soniktrooth** [comment]
I've only ever been able to include svgs by changing the file name to have a `.twig` extension and it works fine. I don't get that error that you are getting @timwasson 
My understanding was that it was a twig limitation, not a Kalastatic one. 
Related: 
It's because of this issue I created this if you're interested: https://github.com/kalamuna/fontawesome-svg-twig

**Comment by thiagodemellobueno** [comment]
@robloach, is this maybe part of the name spacing issue? Can we document the latest greatest?

**Comment by timwasson** [comment]
That's interesting, @soniktrooth... With the above namespace updates I'm able to include SVGs inline with the .svg extension.

**Comment by thiagodemellobueno** [comment]
@timwasson  @RobLoach  - 
I suspect this maybe partially related? 
https://github.com/kalamuna/kalastatic/issues/576

**Comment by RobLoach** [comment]
Strange that Twig wouldn't allow .svg includes. @timwasson Have you found an issue in the twig queue about that?

**Comment by RobLoach** [comment]
Does using `source()` work? https://twig.symfony.com/doc/2.x/functions/source.html
